src/modules/tracking_content/tracking_content.service.ts

```
@@ -602,73 +629,80 @@ export class TrackingContentService {
+              WHERE "courseId" = ANY($1) AND "userId" = ANY($2::uuid[]) AND "tenantId" = $3
+            `;
+
+            const attemptQuery = `
+              SELECT "userId", "courseId", COUNT("assessmentTrackingId") as attempt_count
+              FROM assessment_tracking
+              WHERE "courseId" = ANY($1) AND "userId" = ANY($2::uuid[]) AND "tenantId" = $3
+              GROUP BY "userId", "courseId"
+            `;
+
+            const [certificateResults, attemptResults] = await Promise.all([
+              this.dataSource.query(certificateQuery, [courseIdArray, userIdArray, tenantId]),
```

ESLint

[error] Replace `courseIdArray,·userIdArray,·tenantId` with `⏎················courseIdArray,⏎················userIdArray,⏎················tenantId,⏎··············`

(prettier/prettier)